### DISCUSSION TITLE
Encode query string

### DISCUSSION THREAD

**Discussion by cheng93** [datalust/seq-tickets#1798] [Ideas]
Could we encode the query string?

For example, when filtering by Id we get

http://localhost:5341/#/events?filter=@Id%20%3D%20'event-0fba5e0d29f308db1d42b50000000000'

however it probably should be like

http://localhost:5341/#/events?filter=%40Id%20%253D%20%27event-0fba5e0d29f308db1d42b50000000000%27

**Comment by andymac4182** [comment]
This would help with some apps such as Slack when you past in this URL as well.
